### ISSUE TITLE
TerminalMenu scrolling doesn't work with wrapping entries

### ISSUE THREAD

**Issue by jakobnissen** [JuliaLang/julia#39198]
If you have a terminal shorter than 200 characters, it can be triggered like this:
```
using REPL.TerminalMenus
request(RadioMenu(["a"^150, "b"^200]))
```
, and scrolling up and down a couple of times.
It appears the menus assume all entries fit in one line.

**Comment by timholy** [comment]
Yeah, that's true. In e.g., Cthulhu, care is taken not to exceed a line, but arguably that should be imposed by TerminalMenu itself.
